engine/lib/phx/src/engine/engine.rs

```diff
@@ -145,78 +146,120 @@ impl Engine {
             }
         }
 
+        // Handle suspend/resume
         if let Some(state) = self.window.state {
             match state {
                 WindowState::Suspended => self.winit_window.suspend(),
                 WindowState::Resumed => self.winit_window.resume(),
             }
-
             self.window.state = None;
         }
 
+        // Update title
         if self.window.title != self.cache.window.title {
             self.winit_window
                 .window()
                 .set_title(self.window.title.as_str());
         }
 
+        // === Handle window mode changes ===
         if self.window.mode != self.cache.window.mode {
-            let new_mode = match self.window.mode {
+            match self.window.mode {
                 WindowMode::BorderlessFullscreen => {
-                    Some(winit::window::Fullscreen::Borderless(None))
+                    // Choose the monitor explicitly (primary monitor recommended)
+                    let monitor = self.winit_window.window().primary_monitor();
```

Comment:
If possible, it would be better to detect 'active' monitor where game window is now and use it.
Would be also nice to make it configurable and maybe save user choice when window is dragged to another monitor.
Though there should be another PR for this.